src/Nokia_LCD.h

```diff
@@ -11,6 +11,7 @@
  */
 #pragma once
 #include <Arduino.h>
+#include <SPI.h>
```

Comment:
We should include this header file where it is actually needed, i.e. in the `.cpp` file.

Reply:
fixed